tests/Pyxsim/xmostest_subprocess.py

```diff
@@ -0,0 +1,250 @@
+import platform
```

Comment:
Needs a copyright comment and a comment referencing the license.  See the Source check failures in the Jenkins run.

Reply:
added